pygmt/src/text.py

```diff
@@ -156,7 +156,12 @@ def text_(
     {e}
     {f}
     {h}
-    {i}
+    incols : str
```

Comment:
Current, we alias `-i` to `incols` and need to use `incols="t1"` to select the 2nd word. Here is an example:
```
import pygmt
fig = pygmt.Figure()
fig.basemap(region=[0, 10, 0, 10], projection="X10c", frame=True)
fig.text(x=5, y=5, text="A B C", incols="t1")
fig.show()
```
We can alias `-it` to `incols` and use `incols=1` instead, which is more Pythonic, although it's not backward-compatible. 

Or we can choose a better alias name like `use_word=1`.

Ping @GenericMappingTools/pygmt-maintainers for more thoughts on this.

Reply:
Either way, it looks like we'll need to go through a deprecation cycle? I don't have a strong preference on whether to keep using `incols` for `-it`  or switch to `use_word`. Had a quick look at https://joa-quim.github.io/GMTjl_doc/documentation/text/index.html and doesn't seem like they aliased `it` for `text` so no guidance to follow.

Reply:
@yvonnefroehlich, what do you think is better? Keep using `incols` as the alias (but change it to `-it` instead of `-i`), or use a different alias like `use_word`?

Reply:
> @yvonnefroehlich, what do you think is better? Keep using `incols` as the alias (but change it to `-it` instead of `-i`), or use a different alias like `use_word`?

I think using a different, more descriptive alias, like `use_word`, is better. 

Maybe users (knowing the GMT syntax) are confused, when both `-i` and `-it` are aliased as `incols`. If keeping `incols`, I would stay with `'t`_`word`_`'` as argument. (This were actually my thoughts behind my question in https://github.com/GenericMappingTools/pygmt/issues/1947#issuecomment-1157267031. Thanks @seisman for the nice formulation at the top of this pull request!)
Furthermore, `incols` indicates that the user can specify [numeric] columns, but here a specific word of the whole text in the third column can be specified by passing the word order or position, i. e. for example 0 means not the first column but the first word.

Reply:
I think we will go with `use_word`.

@yvonnefroehlich Please make the changes when you have time.